explainer.md

```diff
@@ -102,7 +103,7 @@ export class NSNet2 {
     this.hiddenSize = 400;
   }
 
-  async load(baseUrl, batchSize, frames) {
+  async build(baseUrl, batchSize, frames) {
```

Comment:
you need to remove async keyword if build method is sync.

Reply:
This is needed, because `buildConstantByNpy` in this function uses `fetch` to download `.npy` files from network which is async.